plugins/scaffolder/src/components/fields/MyGroupsPicker/schema.ts

```diff
@@ -14,42 +14,34 @@
  * limitations under the License.
  */
 
-import { z } from 'zod';
-import { makeFieldSchemaFromZod } from '../utils';
+import { makeFieldSchema } from '@backstage/plugin-scaffolder-react';
 
 /**
  * Field schema for the MyGroupsPicker.
  * @public
  */
 
-export const MyGroupsPickerFieldSchema = makeFieldSchemaFromZod(
-  z.string(),
-  z.object({
-    title: z.string().default('Group').describe('Group'),
-    description: z
-      .string()
-      .default('A group you are part of')
-      .describe('The group to which the entity belongs'),
-  }),
-);
+export const MyGroupsPickerFieldSchema = makeFieldSchema({
+  output: z => z.string(),
```

Comment:
Are we missing the `uiSchema` here?

Reply:
The schema that was there before declared a `default` in the UI schema which, while displayed in the Custom Field Explorer, was not actually picked up in the component and didn't make sense to me. Same for the `description` it declared which AIUI is already "part of" the broader RJSF UI schema. Having eliminated both of these, I omitted the `uiSchema` property since it was marked optional among `makeFieldSchema`'s `options`. Do you agree with my assessment of these individual properties?